gensrc/proto/olap_file.proto

```diff
@@ -93,6 +93,8 @@ message RowsetMetaPB {
     optional int64 num_segments = 22;
     // rowset id definition, it will replace required rowset id 
     optional string rowset_id_v2 = 23;
+    // tablet meta pb, for compaction
+    optional TabletSchemaPB tablet_schema = 24;
```

Comment:
No need, there is already a tablet schema for every segment.

Reply:
There are two reasons that RowsetMeta need the new field. 
1. When compaction, we need to use newset version schema not the schema which newest rowset has. The diff is that when a long load is going, the schema change, and then a short load begin. When the short load was over faster than the long load, the newest schema is the short load rowset has. That's because of that segment don't persist the  schema version too.
2. When we need to use rowset schema，we can read it directly not need to read segment footer.